.github/commitlint.config.mjs

```diff
@@ -8,7 +8,7 @@ const validateBodyMaxLengthIgnoringDeps = async (parsedCommit) => {
       && body != null
       && body.includes('Updates the requirements on');
 
-  const bodyMaxLineLength = 100;
+  const bodyMaxLineLength = 1000;
```

Comment:
I think 1000 is a bit of overkill for a line length, it would make it pretty unreadable.

Reply:
You are right and actually it's a mistake, my mind removed `Line` from `bodyMaxLinelength` so I took the full commit message and I counted the characters (more than 500) instead of counting the characters in the longest line (less than 200). We can go down to 200 I guess, I will open a new PR for that, sorry